kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -68,7 +68,7 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
   private def buildBatch(session: KyuubiBatchSessionImpl): Batch = {
     val batchOp = session.batchJobSubmissionOp
     val batchOpStatus = batchOp.getStatus
-    val batchAppStatus = batchOp.currentApplicationInfo
+    val batchAppStatus = batchOp.getApplicationInfo
```

Comment:
I'm not sure it can alway return a correct info, shall we make this method as `getOrFetchCurrentApplicationInfo` ? Then we in that method, we can call `currentApplicationInfo` if applicationInfo is empty.

Reply:
thanks, done